notebooks/MNIST-torchvision.ipynb

```diff
@@ -0,0 +1,798 @@
+      "text/plain": [
+       "<matplotlib.figure.Figure at 0x7f2d31ecf2e8>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "plot_example(X_pred_cnn, y_pred_cnn[error_mask])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Grid searching parameter configurations\n",
+    "\n",
+    "Finally we want to show an example of how to use sklearn grid search when using torch `Dataset` instances.\n",
+    "\n",
+    "When doing k-fold validation grid search we have the same problem as before that sklearn is only able to do (stratified) splits when the data is sliceable. While skorch knows how to deal with PyTorch `Dataset` objects and only needs `y` to be known beforehand, sklearn doesn't know how to deal with `Dataset`s and needs a wrapper that makes them sliceable.\n",
+    "\n",
+    "Fortunately, we already know that skorch provides such a helper: `SliceDataset`.\n",
+    "\n",
+    "What is left to do is to define our parameter search space and run the grid search with a sliceable instance of `mnist_train`:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 124,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from sklearn.model_selection import GridSearchCV"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 145,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "params = {\n",
+    "    'module__dropout': [0, 0.5, 0.8],\n",
+    "    'max_epochs': [1],\n",
+    "    'verbose': [False],\n",
+    "}"
```

Comment:
You should probably also set `'train_split': [False]` here.

You should probably also explain in one sentence some of the choices. E.g. you probably chose 1 epoch just to make it quicker, not because it is the best choice.